src/msg/async/AsyncConnection.cc

```diff
@@ -803,6 +803,7 @@ void AsyncConnection::tick(uint64_t id)
                                 << " us, fault."
                                 << dendl;
       protocol->fault();
+      logger->inc(l_msgr_faulted_connections);
```

Comment:
agree with @idryomov .. lets use a different counter for idle connections. What about "msgr_idle_connections"